_faq/which-companies-contribute.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Which organizations contribute to Istio?
+order: 51
+type: markdown
+---
+{% include home.html %}
+
+Istio is actively contributed to by dozens of engineers from Google, Ibm, Redhat and others. It also includes Lyft's Envoy.
```

Comment:
Ibm -> IBM
Redhat -> RedHat

I think I'd just say:

Istio is actively contribute to by dozens of engineers from Google, IBM, RedHat, Lyft (via Envoy development), and others.